packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskHelp/TaskHelp.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { inject, observer } from 'mobx-react'
+import { Button, Box } from 'grommet'
+import { Markdownz, Modal, PlainButton } from '@zooniverse/react-components'
+import counterpart from 'counterpart'
+import en from './locales/en'
+
+counterpart.registerTranslations('en', en)
+
+function storeMapper(stores) {
+  const tasks = stores.classifierStore.workflowSteps.activeStepTasks
+  const { isThereTaskHelp } = stores.classifierStore.workflowSteps
+  return {
+    isThereTaskHelp,
+    tasks
+  }
+}
+
+@inject(storeMapper)
+@observer
+class TaskHelp extends React.Component {
+  constructor () {
+    super()
+
+    this.state = {
+      showModal: false
+    }
+  }
+
+  render () {
+    const label = counterpart('TaskHelp.label')
+    const { isThereTaskHelp, tasks } = this.props
+
+    if (isThereTaskHelp) {
+      return (
+        <>
```

Comment:
Out of interest: what does the empty element do here?

Reply:
It's shorthand for React Fragment